packages/code-studio/src/styleguide/Inputs.tsx

```diff
@@ -186,18 +183,23 @@ function Inputs(): React.ReactElement {
 
         <div className="col">
           <form>
-            <h5> Radios </h5>
-            <RadioGroup onChange={handleRadioChange} value={radioValue}>
-              <RadioItem value="1">Toggle this custom radio</RadioItem>
-              <RadioItem value="2">Or toggle this other custom radio</RadioItem>
-              <RadioItem value="3" disabled>
+            <h5 id="inputs-radios-heading">Radios</h5>
+            <RadioGroup
+              aria-labelledby="inputs-radios-heading"
+              onChange={handleRadioChange}
+              value={radioValue}
+            >
+              <Radio value="1">Toggle this custom radio</Radio>
+              <Radio value="2">Or toggle this other custom radio</Radio>
+              <Radio value="3" isDisabled>
                 Disabled radio
-              </RadioItem>
-              <RadioItem value="4" isInvalid>
+              </Radio>
+              <Radio value="4" UNSAFE_className="is-invalid">
```

Comment:
I don't think we should do this in the styleguide - it doesn't seem like a real use case, and we aren't doing it anywhere.
In the Spectrum component, you can set `isInvalid` on the `RadioGroup` itself, not an individual option, which makes more sense IMO - if a single option is invalid, then it should just be disabled.
Also we can show off the error message functionality as well, and description:
```

            <RadioGroup
              aria-labelledby="inputs-radios-heading"
              onChange={handleRadioChange}
              value={radioValue}
              isInvalid={radioValue === '4'}
              description="Select a radio item"
              errorMessage={
                radioValue === '4' ? 'Invalid radio selected' : undefined
              }
            >
              <Radio value="1">Toggle this custom radio</Radio>
              <Radio value="2">Or toggle this other custom radio</Radio>
              <Radio value="3" isDisabled>
                Disabled radio
              </Radio>
              <Radio value="4">Invalid radio</Radio>
              <Radio isHidden={!check4} value="5">
                Extra radio item
              </Radio>
            </RadioGroup>
```

Reply:
Fixed